src/datepicker/ngb-calendar.spec.ts

```diff
@@ -0,0 +1,18 @@
+import {GregorianCalendar} from './ngb-calendar';
+import {NgbDate} from './ngb-date';
+
+describe('ngb-calendar', () => {
+
+  describe('gregorian', () => {
+
+    it('should return today date', () => {
+      const calendar = new GregorianCalendar();
+
+      const jsToday = new Date();
+      const today = new NgbDate(jsToday.getFullYear(), jsToday.getMonth(), jsToday.getDate());
+
+      expect(calendar.getToday().equals(today)).toBeTruthy();
+    });
+  });
+
+});
```

Comment:
You will have to rebase your PR and rename this file to take e348323af37af42e8bec029855775c0c52ae77de into account.